src/dlgProfilePreferences.cpp

```diff
@@ -2860,57 +2860,68 @@ void dlgProfilePreferences::generateDicordTooltips()
 
     auto detail = mudlet->mDiscord.getDetailText(mpHost);
     if (!detail.isEmpty()) {
-        detail = QStringLiteral("(\"%1\")").arg(detail);
+        detail = QStringLiteral("<br/>(\"%1\")").arg(detail);
```

Comment:
Please note **`<br/>`** ***IS NOT VALID HTML!***  That is actually an **XHTML** tag - the one I think you should be using is **`<br>`** - as shown in https://doc.qt.io/qt-5/richtext-html-subset.html - Qt actually states in that link: "*Qt's text widgets are able to display rich text, specified using a subset of HTML 4 markup.*"